### PR TITLE
pay 934 calculate quebec income tax to be withheld for the pay period

### PR DESCRIPTION
**Pull request by mark-h-humi**
https://gethumi.atlassian.net/browse/PAY-934?atlOrigin=eyJpIjoiY2Q4NTJhNjhkMTI5NGQ5ZWI5NmU0OGVmZjNhYjQ4MzEiLCJwIjoiaiJ9

### CODE REVIEW

File: lib/taxman/taxman2023/qc_h.rb [hunk 1]
```diff
@@ -11,7 +11,7 @@ def self.params
     attr_reader(*params)
 
     def amount
-      [(0.06 * qc_d), EMPLOYMENT_INCOME_MAXIMUM_DEDUCTION].min / p
+      [(0.06 * qc_d), (EMPLOYMENT_INCOME_MAXIMUM_DEDUCTION / p)].min
```
Comment (claus-humi) [inline lib/taxman/taxman2023/qc_h.rb:14]: Yeah, this matches the implementation spreadsheet.

File: spec/taxman/taxman2023/tax_calculator_spec.rb [hunk 2]
```diff
@@ -62,4 +63,31 @@
...
+        qc_k1: 0,
+        qc_e1: 17_183_00.to_d,
+        qc_e2: 0,
+        qc_q: 0,
+        qc_q1: 0,
+        qc_l: 0
+      )
+    end
+
+    it "matches the provincial taxes on PDOC/Greg's sheet" do
```
Comment (claus-humi) [inline spec/taxman/taxman2023/tax_calculator_spec.rb:89]: WEBRAS/Greg's sheet?

File: lib/taxman/taxman2023/qc_h.rb (hunk 1 above)
Comment (mark-h-humi) [reply lib/taxman/taxman2023/qc_h.rb:14]: yes and the tax doc is ambiguous

File: spec/taxman/taxman2023/tax_calculator_spec.rb (hunk 2 above)
Comment (mark-h-humi) [reply spec/taxman/taxman2023/tax_calculator_spec.rb:89]: sigh

File: lib/taxman/taxman2023/qc_h.rb (hunk 1 above)
Comment (claus-humi) [reply lib/taxman/taxman2023/qc_h.rb:14]: Yeah, it's really not obvious! Hope Greg's done it right!

**Review by claus-humi — APPROVED** [review]

**Pull request merged**